test/BUILD

```diff
@@ -1,6 +1,21 @@
 package(default_visibility = ["//visibility:public"])
 
-load("//bazel:envoy_build_system.bzl", "envoy_cc_test_library")
+load("//bazel:envoy_build_system.bzl", "envoy_cc_test", "envoy_cc_test_library")
+
+envoy_cc_test(
+    name = "example_configs_test",
+    srcs = [
+        "example_configs_test.cc",
+        "example_configs_test.h",
+    ],
+    data = ["//configs:example_configs"] + ["example_configs_test_setup.sh"],
```

Comment:
Same bazel version. I'm running Xenial inside a VM (AFAIK that doesn't matter for namespacing). Will poke around more today.